### PR TITLE
C2LC-87: Update styling and layout of the Program Block Editor

### PR DESCRIPTION
**Pull request by chosww**
Merged changes for branch C2LC-86: Update styling and layout of the command palette to this commit

### CODE REVIEW

File: src/ProgramBlockEditor.js [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
+import { ReactComponent as DeleteIcon } from './svg/Delete.svg';
+import { ReactComponent as PlayIcon } from './svg/Play.svg';
 import './ProgramBlockEditor.css';
 
 type ProgramBlockEditorProps = {
     intl: any,
     minVisibleSteps: number,
     program: Program,
     selectedAction: SelectedAction,
+    runButtonDisabled: string,
```
Comment (simonbates) [inline src/ProgramBlockEditor.js]: Let's make `runButtonDisabled` into a `boolean `and implement the logic to determine if it's disabled in App.js

    runButtonDisabled={this.state.dashConnectionStatus !== 'connected'}

File: src/EditorContainer.js [hunk 2]
```diff
@@ -10,6 +10,8 @@ type EditorContainerProps = {
     program: Program,
     syntax: TextSyntax,
     mode: EditorMode,
+    runButtonDisabled: string,
```
Comment (simonbates) [inline src/EditorContainer.js]: Let's make `runButtonDisabled` into a `boolean `and implement the logic to determine if it's disabled in App.js

    runButtonDisabled={this.state.dashConnectionStatus !== 'connected'}

File: src/ProgramBlockEditor.js [hunk 3]
```diff
@@ -177,35 +180,51 @@ class ProgramBlockEditor extends React.Component<ProgramBlockEditorProps, {}> {
...
                 <Row>
-                    <Col>
+                    <Col className='ProgramBlockEditor__program-sequence'>
                         {programBlocks}
                     </Col>
                 </Row>
-            </div>
+                <Row className='ProgramBlockEditor__footer'>
+                    <Col>
+                        <Button
```
Comment (simonbates) [inline src/ProgramBlockEditor.js]: Please add an `aria-label`

File: src/App.js [hunk 4]
```diff
@@ -167,62 +162,55 @@ export default class App extends React.Component<{}, AppState> {
...
+                                        onChange={this.handleCommandFromCommandPalette}/>
+                                </div>
+                            </div>
+                        </Col>
+                        <Col md={8} lg={9}>
                             <ProgramBlockEditor
                                 minVisibleSteps={6}
                                 program={this.state.program}
                                 selectedAction={this.state.selectedAction}
+                                runButtonDisabled={this.state.dashConnectionStatus}
```
Comment (simonbates) [inline src/App.js]: Replace with:

    runButtonDisabled={this.state.dashConnectionStatus !== 'connected'}

File: src/ProgramBlockEditor.js [hunk 5]
```diff
@@ -177,35 +180,52 @@ class ProgramBlockEditor extends React.Component<ProgramBlockEditorProps, {}> {
...
+                            aria-label={`${this.props.intl.formatMessage({id:'PlayButton.run'})} ${this.props.program.join(' ')}`}
+                            className='ProgramBlockEditor__run-button'
+                            disabled={this.props.runButtonDisabled !== 'connected'}
```
Comment (simonbates) [inline src/ProgramBlockEditor.js]: Please update to:

    disabled={this.props.runButtonDisabled}

Comment (chosww) [reply src/ProgramBlockEditor.js]: oops!

**Pull request merged**